adoc/admin-disaster-recovery-restore-master-node.adoc

```diff
@@ -0,0 +1,108 @@
+= Recover Lost Master Node
+This chapter describes how to recover {productname} master nodes.
+
+== Replace Single Master Node
+1. Remove the failed master node with skuba.
```

Comment:
Suggest to have a command snippet as an example. The concept is user can copy modify and run w/o issues.